### PR TITLE
Port changes

### LINKED ISSUE #1: Use port 8080 instead of 9090 in production

### LINKED ISSUE #2: Use port 80 instead of port 443 for https requests in production

### LINKED ISSUE #3: Use port 3002 instead of 3000 with the dev environment

### PR DESCRIPTION
**Pull request by FreemRL**
I have changed the dev port to 3002 and the https request port to 80. Fixes #2 and #3

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -11,4 +11,4 @@ services:
     volumes:
       - .:/app
     ports:
-      - 3000:3000
+      - 3000:3002
```
Comment (AymanAzzam) [inline docker-compose.yaml]: hint:
- host-port:docker-port

so it should be 3002:3000 not 3000:3002

the same in the previous file with port 80

**Review by AymanAzzam — CHANGES_REQUESTED** [review]
hint:

host-port:docker-port

File: docker-compose.yaml (hunk 1 above)
Comment (FreemRL) [reply docker-compose.yaml]: Sorry man I actually got it right first but changed it wrong. I changed the wrong order now.

**Comment by AymanAzzam** [comment]
@FreemRL well done

**Pull request merged**